src/borough/borough.controller.ts

```diff
@@ -44,4 +49,24 @@ export class BoroughController {
       params.boroughId,
     );
   }
+
+  @Get("/:boroughId/community-districts/:communityDistrictId/capital-projects")
+  async findCapitalProjectsByBoroughIdCommunityDistrictId(
+    @Param(
```

Comment:
Oh, interesting. The way we solved this on Horatio's branch was to merge the definitions. Merging the defintions fits the prefixed pipes pattern better. However, it kinda waters down the schema definitions because it relies on "partial".

Using the pipe in each type of parameter might actually be more powerful because it uses the zod schema definition as-is

https://github.com/NYCPlanning/ae-zoning-api/blob/3a2ef3808bfd802fe4751b97aedbbbbdcfd39791/src/city-council-district/city-council-district.controller.ts#L57

Reply:
We might want to update CCD to directly use pipe in the param decorators so that both endpoints are consistent.

Reply:
https://github.com/NYCPlanning/ae-zoning-api/issues/340